todos/week3/4_classestodo.py

```diff
@@ -1,12 +1,14 @@
 """
 Creating classes for your classmates
 """
-
+# TODO: #############################################################################
+# TODO: Section 1:
 # TODO: Create a class for an Employee, and include basic data
 # TODO: like hours worked, salary, first name, last name, age, and title
 
 
-
+# TODO: ##############################################################################
+# TODO: Section 2:
```

Comment:
Section 2 should not have its own section. Rather, it should be included in step 1 as a sub-problem because you would not want someone to have large sections of comments in the middle of their class definition.